Wikipedia/assets/index.js

```diff
@@ -337,11 +337,7 @@ var transformer = require("../transformer");
 var utilities = require("../utilities");
 
 function shouldAddImageOverflowXContainer(image) {
-    if ((image.width > (window.screen.width * 0.8)) && !utilities.isNestedInTable(image)){
-        return true;
-    }else{
-        return false;
-    }
+    return ((image.width > (window.screen.width * 0.8)) && !utilities.isNestedInTable(image)) ? true : false;
```

Comment:
can probably just `return ((image.width > (window.screen.width * 0.8)) && !utilities.isNestedInTable(image));` unless some permutation of that doesn't return a boolean in js